model/database/registry.yaml

```
@@ -114,7 +114,15 @@ groups:
 
           This attribute has stability level RELEASE CANDIDATE.
         examples: ["102", "ORA-17002", "08P01", "404"]
+      - id: db.response.row_count
+        type: int
+        stability: experimental  # RELEASE CANDIDATE
```

this will need to stabilize after the core set, let's not add it yet

```suggestion
        stability: experimental
```